src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -68,14 +65,15 @@ class JSONSerializer(Serializer):
     +-------------------------------------+----------+----------------------------------------+
     |                           |          | Callable(SerializationContext, str) -> str       |
     |                           |          |                                                  |
-    | ``subject.name.strategy`` | callable | Instructs the JsonSerializer on how to construct |
-    |                           |          | Schema Registry subject names.                   |
+    | ``subject.name.strategy`` | callable | Defines how Schema Registry subject names are    |
+    |                           |          | constructed.                                     |
     |                           |          | Defaults to topic_subject_name_strategy.         |
     +---------------------------+----------+--------------------------------------------------+
 
-    Schemas are registered to namespaces known as Subjects which define how a
-    schema may evolve over time. By default the subject name is formed by
-    concatenating the topic name with the message field separated by a hyphen.
+    Schemas are registered against subject names in Confluent Schema Registry
+    that define a scope in which the schemas can be evolved. By default, the subject
+    name is formed by concatenating the topic name with the message field separated
```

Comment:
Message field is vague. Maybe just add `message field ("key" or "value") ...`

Reply:
good point.